src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -121,6 +125,9 @@ void fillInnerParts() {
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+
+        viewListPanel = new PersonListPanel(logic.getViewedPersonList()); //TODO: CHANGE THIS!!!!!!
```

Comment:
Remember to remove the comment later on!

Reply:
okay! need to do this modification anyway before submitting the project